### ISSUE TITLE
Bot Not Recognizing Lat/Long - Resulted In Ban

### ISSUE THREAD

**Issue by chromeplatedhrt** [Novalys/PokemonGo-Bot-SimpleGUI#81]
When I start the bot and put in local lat/long it is sending me London, England no matter what I put in. Needless to say I have been soft banned... I have tried with both my PTC and my Google and will enter a lat/long and hit set location and it resets back to default. I didn't notice at 1st and it resulted int he bans.

**Comment by Novalys** [comment]
Hello @chromeplatedhrt, quick question you were using the release binary right? I'm sorry if you are, the latest code is in the source itself and the binary is not up to date, I wasn't available during the weekend to create the latest binary. Today I'm planning on catching up with development and will upload also the latest binary.

**Comment by chromeplatedhrt** [comment]
@Novalys I was using the newest under releases. I used it Thursday, Friday and Saturday, I believe, and it was working well! Not sure what happened. Hopefully the ban won't last.  Should I have downloaded something different? I am using a different PC than I was over the weekend. 
I also have the issue with having more than one Google account and it will only let me use one of them, but I believe it was marked as a bug and the dev that responded was going to add a reset token button.
